src/main/kotlin/cacophony/controlflow/functions/GenerateCall.kt

```diff
@@ -66,10 +72,15 @@ fun generateCall(
     }
 
```

Comment:
We pushed necessary arguments to the stack, but [our call convention](https://github.com/mhorod/cacophony/blob/main/DEVELOPMENT.md#function-call-and-return) says that we want to reserve space for results that doesn't fit into single register, and as I unterstand your implementation our EpilogueHandler you use it here and move return values into corresponding places in memory.
I think we need to reserve this space before the call here